docs/docs-content/clusters/pcg/manage-pcg/registry-override.md

```diff
@@ -0,0 +1,195 @@
+---
+sidebar_label: "Override Registry Configuration"
+title: "Override Registry Configuration"
+description: "Learn how to override the image registry configuration for a Private Cloud Gateway (PCG) in Palette."
+hide_table_of_contents: false
+sidebar_position: 60
+tags: ["pcg"]
+---
+
+You can override the image registry configuration for a Private Cloud Gateway (PCG) to use a custom image registry for
+the PCG cluster. This feature is useful when you want to use a custom image registry to store and manage the images used
```

Comment:
```suggestion
You can override the image registry configuration for a Private Cloud Gateway (PCG) to use a custom image registry. This feature is useful when you want to use a custom image registry to store and manage the images used
```

Reply:
_for the PCG cluster_ seems a bit redundant in terms of copy because you mention it in the following sentence, but if it's essential to the technical context, makes sense to keep it 🙌🏻